shell/src/main/java/alluxio/cli/fs/command/CheckConsistencyCommand.java

```diff
@@ -54,7 +72,18 @@ public CheckConsistencyCommand(FileSystemContext fsContext) {
   @Override
   protected void runPlainPath(AlluxioURI plainPath, CommandLine cl)
       throws AlluxioException, IOException {
-    runConsistencyCheck(plainPath, cl.hasOption("r"));
+    int threads;
+    try {
+      threads = cl.hasOption(THREADS_OPTION.getOpt())
+          ? Integer.parseInt(cl.getOptionValue(THREADS_OPTION.getOpt())) :
+          Runtime.getRuntime().availableProcessors() * 2;
+      if (threads < 1) {
+        throw new IOException(PARSE_THREADS_FAILURE_MSG);
```

Comment:
in the message, can we include what `threads` was parsed as?

Reply:
:+1: